vms/components/multisig/camino_multisig.go

```diff
@@ -17,16 +17,19 @@ import (
 // MaxMemoSize is the maximum number of bytes in the memo field
 const MaxMemoSize = 256
 
-var errMemoIsToBig = errors.New("msig alias memo is to big")
-
 var (
 	_ snow.ContextInitializable = (*Alias)(nil)
 	_ verify.Verifiable         = (*Alias)(nil)
+
+	errMemoIsToBig = errors.New("msig alias memo is to big")
```

Comment:
```suggestion
	errMemoIsTooBig = errors.New("msig alias memo is too big")
```
Typo here, and also repeated along the changes.